dataeng/resources/snowflake-schema-builder.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -ex
+
+# Setup
+cd $WORKSPACE/warehouse-transforms
+pip install -r tools/dbt_schema_builder/requirements.txt
+
+# Create a new git branch
+now=$(date +%d_%m_%Y_%H_%M_%S)
+branchname="builder_$now"
+git checkout -b "$branchname"
+
+# Run the dbt script to update schemas and sql
+python tools/dbt_schema_builder/schema_builder.py build --schemas $SCHEMAS --project $DBT_PROJECT --profile $DBT_PROFILE --target $DBT_TARGET --profiles-dir $WORKSPACE/analytics-secure/analytics-tasks/
+
+# Commit all changes to the new branch
+git commit -a --message "Schema Builder automated dbt update at $now"
```

Comment:
I assume that most of the time we won't expect to see differences.   Is there something we should do to check for that condition, and not create a PR?   Or will there be dates and such in various comments that will appear and churn every time this is run?

Reply:
Agreed - could `git status` be used to see if the commit was empty?

Reply:
Makes sense, but it will introduce a bit of work to get the schema files to sort deterministically in the generation script (to make the comparisons work). Hopefully I can just fix it in the query. 🤞

Reply:
I've updated the generation script and just pushed changes here to address this. I've tested it as well as I can, adding, deleting, and changing files.